src/tfvc/scm/model.ts

```diff
@@ -137,10 +146,26 @@ export class Model {
             }
         });
 
-        this._mergeGroup = new MergeGroup(merge);
+        this._conflictsGroup = new ConflictsGroup(conflicts);
         this._includedGroup = new IncludedGroup(included);
         this._excludedGroup = new ExcludedGroup(excluded);
 
         this._onDidChange.fire(this.Resources);
     }
+
+    private conflictMatchesPendingChange(change: IPendingChange, conflict: IConflict): boolean {
+        let result: boolean = false;
+        if (change && change.localItem && conflict && conflict.localPath) {
+            // TODO: If resource or conflict are renames we have a lot more work to do
+            //       We are postponing this work for now until we have evidence that it happens a lot
```

Comment:
Would we want to send telemetry here as well?  Or is the comment above at the location where we'd track this?